template.rb

```diff
@@ -84,6 +96,45 @@
 end
 CODE
 
+if yes?("Are you hosting Shipit on Heroku? (y/n)")
+  inject_into_file "Procfile", " -p $PORT", after: "web: bundle exec rails s"
+  inject_into_file "Gemfile", "ruby '2.2.3'", after: "source 'https://rubygems.org'\n"
```

Comment:
Good point. No idea what would be best. We could use whatever version was used to execute this template.